source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -34,8 +34,14 @@ public class ForestGameArea extends GameArea {
   private static final int NUM_GHOSTS = 0;
   private static final int NUM_GRUNTS = 5;
   private static final int NUM_BOSS = 4;
-  
-  
+
+
+  private static final int NUM_BOSSKING2=3;
+  private static final int NUM_BOSSKING1=1;
+
+  private int wave = 0;
+  private Timer waveTimer;
+
   private Timer bossSpawnTimer;
   private int bossSpawnInterval = 10000; // 1 minute in milliseconds
```

Comment:
This is not one minute, this is 10 seconds

Reply:
we didnt use that variable it was for a experimental thing